<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Figaro Editor</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>130</y>
      <width>221</width>
      <height>471</height>
     </rect>
    </property>
    <property name="acceptDrops">
     <bool>false</bool>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tabTree">
     <attribute name="title">
      <string>Map Tree</string>
     </attribute>
     <widget class="QTreeWidget" name="projectTree">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>221</width>
        <height>431</height>
       </rect>
      </property>
      <attribute name="headerVisible">
       <bool>false</bool>
      </attribute>
      <column>
       <property name="text">
        <string notr="true">1</string>
       </property>
      </column>
     </widget>
    </widget>
    <widget class="QWidget" name="tabTiles">
     <attribute name="title">
      <string>Tiles</string>
     </attribute>
     <widget class="QGraphicsView" name="tilemapView">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>221</width>
        <height>311</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>NPC</string>
     </attribute>
     <widget class="QTableWidget" name="npcTableWidget">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>211</width>
        <height>121</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>211</width>
        <height>0</height>
       </size>
      </property>
      <attribute name="horizontalHeaderMinimumSectionSize">
       <number>60</number>
      </attribute>
      <attribute name="verticalHeaderMinimumSectionSize">
       <number>30</number>
      </attribute>
      <row>
       <property name="text">
        <string>Name</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>X</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>Y</string>
       </property>
      </row>
      <column>
       <property name="text">
        <string>Value</string>
       </property>
      </column>
     </widget>
    </widget>
   </widget>
   <widget class="QGraphicsView" name="mapView">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>761</width>
      <height>601</height>
     </rect>
    </property>
    <property name="acceptDrops">
     <bool>false</bool>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOn</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOn</enum>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>10</y>
      <width>221</width>
      <height>121</height>
     </rect>
    </property>
    <property name="title">
     <string>Layers</string>
    </property>
    <widget class="QComboBox" name="cmbLayers">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>20</y>
       <width>221</width>
       <height>27</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>Background Layer</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Solid Layer</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Foreground Layer</string>
      </property>
     </item>
    </widget>
    <widget class="QCheckBox" name="chkBackground">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>50</y>
       <width>151</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Background Layer</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="chkSolid">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>70</y>
       <width>111</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Solid Layer</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="chkForeground">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>90</y>
       <width>151</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Foreground Layer</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="chkOpacityBackground">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>50</y>
       <width>51</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="resources/resources.qrc">
       <normaloff>:/icons/onion.png</normaloff>:/icons/onion.png</iconset>
     </property>
    </widget>
    <widget class="QCheckBox" name="chkOpacitySolid">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>70</y>
       <width>51</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="resources/resources.qrc">
       <normaloff>:/icons/onion.png</normaloff>:/icons/onion.png</iconset>
     </property>
    </widget>
    <widget class="QCheckBox" name="chkOpacityForegrond">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>90</y>
       <width>51</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="resources/resources.qrc">
       <normaloff>:/icons/onion.png</normaloff>:/icons/onion.png</iconset>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionSelector"/>
   <addaction name="actionPencil"/>
   <addaction name="actionPaintCan"/>
   <addaction name="actionEraser"/>
   <addaction name="actionMagicEraser"/>
   <addaction name="actionInformation"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1024</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMap">
    <property name="title">
     <string>Map</string>
    </property>
    <addaction name="mpActionNew_Map"/>
    <addaction name="mpActionMap_Properties"/>
   </widget>
   <widget class="QMenu" name="menuProject">
    <property name="title">
     <string>Project</string>
    </property>
    <addaction name="prActionSet_Main_Folder"/>
    <addaction name="prActionEngine_Config"/>
    <addaction name="prActionSave"/>
   </widget>
   <widget class="QMenu" name="menuNPC">
    <property name="title">
     <string>NPC</string>
    </property>
    <addaction name="npcActionAdd_NPC"/>
    <addaction name="actionRemove_Current_NPC"/>
   </widget>
   <addaction name="menuProject"/>
   <addaction name="menuMap"/>
   <addaction name="menuNPC"/>
  </widget>
  <action name="tlActionImportTileset">
   <property name="icon">
    <iconset resource="resources/resources.qrc">
     <normaloff>:/icons/addTiles.png</normaloff>:/icons/addTiles.png</iconset>
   </property>
   <property name="text">
    <string>Import Tileset</string>
   </property>
  </action>
  <action name="dbActionImport_test_Tileset">
   <property name="text">
    <string>Import test Tileset</string>
   </property>
  </action>
  <action name="mpActionNew_Map">
   <property name="icon">
    <iconset resource="resources/resources.qrc">
     <normaloff>:/icons/addMap.png</normaloff>:/icons/addMap.png</iconset>
   </property>
   <property name="text">
    <string>New Map</string>
   </property>
  </action>
  <action name="actionFile">
   <property name="text">
    <string>File</string>
   </property>
  </action>
  <action name="prActionSave">
   <property name="icon">
    <iconset resource="resources/resources.qrc">
     <normaloff>:/icons/save.png</normaloff>:/icons/save.png</iconset>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="prActionEngine_Config">
   <property name="icon">
    <iconset resource="resources/resources.qrc">
     <normaloff>:/icons/config.png</normaloff>:/icons/config.png</iconset>
   </property>
   <property name="text">
    <string>Engine Config</string>
   </property>
  </action>
  <action name="prActionSet_Main_Folder">
   <property name="icon">
    <iconset resource="resources/resources.qrc">
     <normaloff>:/icons/mainFolder.png</normaloff>:/icons/mainFolder.png</iconset>
   </property>
   <property name="text">
    <string>Set Main Folder</string>
   </property>
  </action>
  <action name="actionLoad_Existing_Map">
   <property name="text">
    <string>Load Existing Map</string>
   </property>
  </action>
  <action name="actionPencil">
   <property name="icon">
    <iconset resource="resources/resources.qrc">
     <normaloff>:/icons/pencil.png</normaloff>:/icons/pencil.png</iconset>
   </property>
   <property name="text">
    <string>pencil</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
  </action>
  <action name="actionEraser">
   <property name="icon">
    <iconset resource="resources/resources.qrc">
     <normaloff>:/icons/eraser.png</normaloff>:/icons/eraser.png</iconset>
   </property>
   <property name="text">
    <string>eraser</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+E</string>
   </property>
  </action>
  <action name="actionPaintCan">
   <property name="icon">
    <iconset resource="resources/resources.qrc">
     <normaloff>:/icons/paintcan.png</normaloff>:/icons/paintcan.png</iconset>
   </property>
   <property name="text">
    <string>paintCan</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="mpActionMap_Properties">
   <property name="icon">
    <iconset resource="resources/resources.qrc">
     <normaloff>:/icons/mapProperties.png</normaloff>:/icons/mapProperties.png</iconset>
   </property>
   <property name="text">
    <string>Edit Map Properties</string>
   </property>
  </action>
  <action name="actionInformation">
   <property name="icon">
    <iconset resource="resources/resources.qrc">
     <normaloff>:/icons/information.png</normaloff>:/icons/information.png</iconset>
   </property>
   <property name="text">
    <string>information</string>
   </property>
  </action>
  <action name="actionMagicEraser">
   <property name="icon">
    <iconset resource="resources/resources.qrc">
     <normaloff>:/icons/magicEraser.png</normaloff>:/icons/magicEraser.png</iconset>
   </property>
   <property name="text">
    <string>magicEraser</string>
   </property>
  </action>
  <action name="npcActionAdd_NPC">
   <property name="icon">
    <iconset resource="resources/resources.qrc">
     <normaloff>:/icons/addNPC.png</normaloff>:/icons/addNPC.png</iconset>
   </property>
   <property name="text">
    <string>Add New NPC</string>
   </property>
  </action>
  <action name="actionSelector">
   <property name="icon">
    <iconset resource="resources/resources.qrc">
     <normaloff>:/icons/cursor.png</normaloff>:/icons/cursor.png</iconset>
   </property>
   <property name="text">
    <string>Selector</string>
   </property>
  </action>
  <action name="actionRemove_Current_NPC">
   <property name="icon">
    <iconset resource="resources/resources.qrc">
     <normaloff>:/icons/removeNPC.png</normaloff>:/icons/removeNPC.png</iconset>
   </property>
   <property name="text">
    <string>Remove Current NPC</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="resources/resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>mpActionNew_Map</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>onCreateNewMapAction()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>283</x>
     <y>203</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>prActionSave</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>onSaveAction()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>283</x>
     <y>203</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>prActionEngine_Config</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>onGeneralConfigAction()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>283</x>
     <y>203</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>prActionSet_Main_Folder</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>onSetMainFolderAction()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>416</x>
     <y>269</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionPencil</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>onPencilToolbar()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>416</x>
     <y>269</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionEraser</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>onEraserToolbar()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>416</x>
     <y>269</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionPaintCan</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>onPaintCanToolbar()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>416</x>
     <y>269</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>mpActionMap_Properties</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>onMapPropertiesAction()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>416</x>
     <y>281</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionInformation</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>onInformationToolbar()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>416</x>
     <y>281</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionMagicEraser</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>onMagicEraserToolbar()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>511</x>
     <y>349</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>npcActionAdd_NPC</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>onAddNPC()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>511</x>
     <y>349</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionSelector</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>onSelectorToolbar()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>511</x>
     <y>349</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>onImportTilesetAction()</slot>
  <slot>debug_onImportTestTilesetAction()</slot>
  <slot>onCreateNewMapAction()</slot>
  <slot>onSaveAction()</slot>
  <slot>onGeneralConfigAction()</slot>
  <slot>onSetMainFolderAction()</slot>
  <slot>onPencilToolbar()</slot>
  <slot>onEraserToolbar()</slot>
  <slot>onPaintCanToolbar()</slot>
  <slot>onMapPropertiesAction()</slot>
  <slot>onInformationToolbar()</slot>
  <slot>onMagicEraserToolbar()</slot>
  <slot>onAddNPC()</slot>
  <slot>onSelectorToolbar()</slot>
 </slots>
</ui>
